folderAWG/Interoperability Precis.md

```diff
@@ -0,0 +1,128 @@
+● Reuse conversational capabilities from other developers, including those running on other conversational platforms 
+● Share dialog, data, context, and controls with other voice agents, regardless of platform or technological parentage 
+● Be addressable directly through an explicit user request 
+● Make itself available to meet a user request on behalf of another trusted voice agent ● Support security and privacy by design. 
+
+### VI. The Open Voice Network Interoperability Initiative: From Talk to Action 
+To develop the standard message formats, protocols, and data formats necessary for agent-to-agent interoperability, the Open Voice Network will cut the issue into layers. Presently, we envision six layers, termed “enhancements,” that operate atop a standardized communication infrastructure. 
+This infrastructure will include: 
+● a set of well-defined messages for starting, stopping, suspending, and resuming voice agents; 
+● protocols consisting of sequences of messages that implement the enhancement; and, ● a set of self-defined data formats for transferring data among agents. 
+The six enhancements are as follows: 
+1. Remote invocation and location of a second voice agent, regardless of platform. 
+2. Connection to a specific agent through an explicit request. Such a request could contain the registered name of a voice agent. The Open Voice Network envisions a “Voice Registration System” (VRS) that allows an owner of voice agents to uniquely register the name of the agent, and enables a direct connection to that agent when that agent is explicitly requested by the user. 
+3. Switching between voice agents. To fulfill user needs, a voice agent may need to switch the user to a second voice agent—and then back again. For instance, an experienced 
+traveler may wish to switch back-and-forth between an airline-specific agent, a hotel chain-specific agent, and the agents of restaurants. 
+4. Processing implicit requests. We define “implicit” as a request that searches across voice agent services rather than seeking specific, named voice agent destination. This will rely upon an arbitration service to select (or recommend) a voice agent from many. 
+5. Sharing data and context between voice agents. This enhancement advances #3, above, and enables agents to manage the sharing of data, context, and controls. It also will enable agents to call upon the universe of conversational capabilities, regardless of platform parentage. 
+6. Generating and operating a private agent persona. A persona refers to the voice and characteristics presented by a conversational agent. Such agents have a distinctive sound and ambiance that leads to branding, the promotion of a particular enterprise by means of advertising, and distinctive design. This enhancement will enable an enterprise voice agent to fully express a specific brand tone, and enable capabilities to adapt to or adopt the persona of the enterprise agent. 
+Other possible enhancements include: 
```

Comment:
Without an extra return here, this line will appear in line with the previous paragraph.